lms/static/js/student_account/views/account_settings_factory.js

```diff
@@ -187,9 +188,83 @@
+                                    'facebook.com/[your_username].'
+                                ),
+                                persistChanges: true,
+                                placeholder: gettext('ex: www.facebook.com/edX')
+                            }),
+                        },
+                        {
+                            view: new AccountSettingsFieldViews.TextFieldView({
+                                model: userAccountModel,
+                                title: gettext('Twitter Link'),
+                                valueAttribute: 'twitter_link',
+                                helpMessage: gettext(
+                                    'Optionally, link to your Twitter profile from your edX learner profile ' +
+                                    'by adding the URL to your personal page or your username, found at ' +
+                                    'twitter.com/[your_username].'
+                                ),
+                                persistChanges: true,
+                                placeholder: gettext('ex: www.twitter.com/edXOnline')
+                            }),
+                        },
+                        {
+                            view: new AccountSettingsFieldViews.TextFieldView({
+                                model: userAccountModel,
+                                title: gettext('LinkedIn Link'),
+                                valueAttribute: 'linkedin_link',
+                                helpMessage: gettext(
+                                    'Optionally, link to your LinkedIn profile from your edX learner profile ' +
+                                    'by adding the URL to your personal page or your username, found at ' +
+                                    'linkedin.com/in/[your_username].'
+                                ),
+                                persistChanges: true,
+                                placeholder: gettext('ex: www.linkedin.com/in/agarwaledu/')
+                            }),
+                        },
+                    ]
                 }
             ];
 
+            // Add the social link fields
+            var socialFields = {
+                title: gettext('More Social Links'),
+                fields: []
+            }
+
+            for (var social_platform in socialPlatforms) {
+                var platformData = socialPlatforms[social_platform];
+                socialFields.fields.push(
+                    {
+                        view: new AccountSettingsFieldViews.SocialLinkTextFieldView({
+                            model: userAccountModel,
+                            title: gettext(platformData['display_name'] + ' Link'),
+                            valueAttribute: 'social_link',
+                            helpMessage: gettext(
+                                'Optionally, link to your ' + platformData['display_name'] +
```

Comment:
With the "Optionally" part extracted to the help text under the "Social Media Links" title, this help text can be shortened: "Enter the URL to your ```platformData[`display_name`]``` profile, using your ```display_name``` username.